pkg/branding/brandedtext_test.go

```diff
@@ -0,0 +1,64 @@
+package branding
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	brandedProductNameRHACS    = "Red Hat Advanced Cluster Security for Kubernetes"
+	brandedProductNameStackrox = "StackRox"
+)
+
+var _ suite.TearDownTestSuite = (*BrandedTextTestSuite)(nil)
```

Comment:
You can add a type assertion for SetupTest as well if you want. [This is the interface](https://pkg.go.dev/github.com/stretchr/testify/suite#SetupTestSuite).

Reply:
Correction, you'd need the SetupTestSuite: [this is the correct interface](https://pkg.go.dev/github.com/stretchr/testify/suite#SetupAllSuite).

Reply:
Ah, thank you. I didn't use the one for SetupTest() since I have not implemented it, but I am using SetupSuite() and found the corresponding suite.SetupAllSuite in the documentation